pallets/liquidity-pools-gateway/routers/src/lib.rs

```diff
@@ -219,127 +185,3 @@ pub struct FeeValues {
 	/// The gas limit used when executing the EVM call.
 	pub gas_limit: U256,
 }
-
-/// A generic router used for executing XCM calls.
-#[derive(Debug, Encode, Decode, Clone, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
-pub struct XCMRouter<T: pallet_xcm_transactor::Config> {
```

Comment:
Given that we "might" use xcm in the future, would it make sense to keep this here and minimally maintain this when doing polkadot version updates etc.? I'm thinking that re-doing an xcm setup from scratch might be even more troublesome later on.

Reply:
In the incoming refactor, this `routers` module will disappear. We no longer can have "outbound" routers without an "inbound" router. In case we need this logic to build a bidirectional XCM router in the future, we can just look in this PR and recover that logic to build that new router.

Note that XCM is still used by the chain, just that not by the that router.